src/main/java/com/leetcode/practice/easy/binary_search/OccuranceOfElement.java

```diff
@@ -0,0 +1,45 @@
+package com.leetcode.practice.easy.binary_search;
+//To find frequency of target element.
+    public class OccuranceOfElement {
```

Comment:
OccuranceOfElement is a verb. Class name should always be nouns. So move these methods under this class to BinarySearch